crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -1497,6 +1498,9 @@ pub enum ReadError {
         key: Cow<'static, str>,
         cause: String,
     },
+
+    #[error("The device certificate cannot be read from the path: {path} ")]
+    CertNotFound { path: Utf8PathBuf },
```

Comment:
I'm not happy to list this `CertNotFound` as tedge config's `ReadError`. I feel it's too specific.